test/support/integration/plugins/modules/timezone.py

```diff
@@ -126,7 +126,7 @@ def __new__(cls, module):
                     module.fail_json(msg='Adjusting timezone is not supported in Global Zone')
 
             return super(Timezone, SmartOSTimezone).__new__(SmartOSTimezone)
-        elif re.match('^Darwin', platform.platform()):
+        elif platform.system() == 'Darwin':
```

Comment:
Since this is a module change, should this be in a separate PR so it can be backported?

Reply:
I'd rather just leave it here, and wait until the upstream PR is merged before backporting. We should probably not cherry-pick this anyway until the upstream PR is merged: https://github.com/ansible-collections/community.general/pull/945

2.9 tests will likely have to be updated further than devel if we backport the macos tests.

Reply:
I think the intention was to only backport this to `stable-2.10`. We'll continue testing `stable-2.8` and `stable-2.9` with OS X 10.11.

Reply:
I'd just leave this as is then.  This is just testing support.